packages/backend/src/rapyd/documents/documents.service.ts

```diff
@@ -0,0 +1,40 @@
+import logger from '../../utils/logger'
+import { Request, Response } from 'express'
+import { BaseResponse } from '../../shared/models/BaseResponse'
+import { makeRapydGetRequest } from '../utills/request'
+
+const log = logger('rapydService')
+
+export const getDocumentTypes = async (
+  req: Request,
+  res: Response<BaseResponse>
+) => {
+  try {
+    const { country } = req.query
```

Comment:
maybe verify the existance of country inside the query object aswell before actually making the http request further down, so you treat the case of sending a country= (nothing) query param to rapyd, which would probably still return something but it would be unexpected